src/modules/AieraChat/index.tsx

```diff
@@ -39,10 +41,12 @@ export function AieraChat(): ReactElement {
     const virtuosoRef = useRef<VirtuosoMessageListMethods<ChatMessage>>(null);
 
     // Set up Ably realtime client
-    const { createAblyRealtimeClient } = useAbly();
+    const { createAblyRealtimeClient, subscribeToChannel, unsubscribeFromChannel } = useAbly();
     const [clientReady, setClientReady] = useState(false);
     const initializingRef = useRef(false);
     const clientRef = useRef<Ably.Realtime | null>(null);
+    const [subscribedTitleChannel, setSubscribedTitleChannel] = useState<RealtimeChannel | undefined>(undefined);
```

Comment:
Should this be a ref instead since we're only using it to store the last value and it's not added to the `useEffect` dependency array?